### PR TITLE
table styling for all except discussion

### PR DESCRIPTION
**Pull request by drouba**

### CODE REVIEW

File: public/discussion_list.php [hunk 1]
```diff
@@ -7,18 +7,16 @@
...
                     <th>Created At</th>
                 </tr>
             </thead>
             <tbody>
         <?php foreach ($discussions as $row) { ?>
             <tr>
-                <td><?php echo escape($row->id); ?></td>
                 <td><a href="discussion.php?id=<?php echo $row->id; ?>"><?php echo escape($row->title); ?></a></td>
-                <td><?php echo escape($row->user_id); ?></td>
+                <td><?php echo escape($row->user->first_name." ".$row->user->last_name); ?></td>
```
Comment (jamestjw) [inline public/discussion_list.php:19]: hey didnt we implement  get_full_name()

File: public/view_lecture_teams.php [hunk 2]
```diff
@@ -50,22 +50,20 @@
...
                         <th>Email Address</th>
-                        <th>Created At</th>
                     </tr>
                 </thead>
                 <tbody>
                 <?php $students_in_team = $row->team_members;
                 foreach ($students_in_team as $student) {?>
                 <tr>
-                    <td><?php echo escape($student->user_id); ?></td>
+                    <td><?php echo escape($student->user->student_id); ?></td>
```
Comment (jamestjw) [inline public/view_lecture_teams.php:63]: can you help add the `includes` for users as well 
image

File: public/view_users.php [hunk 3]
```diff
@@ -24,23 +24,23 @@
...
             </thead>
             <tbody>
         <?php foreach ($result as $row) { ?>
             <tr>
-                <td><?php echo escape($row->id); ?></td>
                 <td><?php echo escape($row->first_name); ?></td>
                 <td><?php echo escape($row->last_name); ?></td>
                 <td><?php echo escape($row->email); ?></td>
-                <td><?php echo escape($row->password_digest); ?></td>
+                <td><?php echo ( $row->is_instructor == true ? "yes" : "no"); ?></td>
```
Comment (jamestjw) [inline public/view_users.php:42]: `echo ( $row->is_instructor ? "yes" : "no");`

don't think you need the == true

**Review by jamestjw — CHANGES_REQUESTED** [review]

File: public/discussion_list.php (hunk 1 above)
Comment (drouba) [reply public/discussion_list.php:19]: oh that's right I implemented that! lol will change it

File: public/view_users.php (hunk 3 above)
Comment (drouba) [reply public/view_users.php]: done

File: public/view_lecture_teams.php (hunk 2 above)
Comment (drouba) [reply public/view_lecture_teams.php:63]: done

File: public/discussion_list.php (hunk 1 above)
Comment (drouba) [reply public/discussion_list.php]: done

**Review by jamestjw — APPROVED** [review]

**Pull request merged**